docs/reference/quickstart/run-elasticsearch-locally.asciidoc

```diff
@@ -125,6 +125,10 @@ The service is started with a trial license. The trial license enables all featu
 
 To connect to the {es} cluster from a language client, you can use basic authentication with the `elastic` username and the password you set in the environment variable.
 
+.*Expand* for examples
+[%collapsible]
+==============
+
 You'll use the following connection details:
```

Comment:
the collapsible title indicates to me that you're just hiding examples, but it looks like you're also hiding the default endpoint and auth details here

Reply:
ah 💯 I'll reword the text because I actually want to clear as much real estate as possible on this page :)

Reply:
@shainaraskas do you think we could get away with pulling out this text from intro too?

Feels like excessive text is clogging up reader's working memory. Want this to really be about copying and running the ~5 commands.

**When would I use this setup?**

Use this setup if you want to quickly spin up Elasticsearch (and Kibana) for local development or testing.

~~For example you might:~~

~~Want to run a quick test to see how a feature works.~~  
~~Follow a tutorial or guide that requires an Elasticsearch cluster, like our [quick start guide](https://elasticsearch_bk_112355.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/getting-started.html).~~  
~~Experiment with the Elasticsearch APIs using different tools, like the Dev Tools Console, cURL, or an Elastic programming language client.~~  
~~Quickly spin up an Elasticsearch cluster to test an executable [Python notebook](https://github.com/elastic/elasticsearch-labs/tree/main/notebooks#readme) locally.~~

Reply:
I like it. people don't have to read it if they don't care - we have scannable headings on the page they can use to jump to the part they are interested in. 

personally, I don't agree with hiding content behind a collapsible element, but since it pops open when we search it's ok